component/admin/views/payment_configurations/tmpl/test.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
nah, it is used to view display of payment gateway and to send sandboxed payment